nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
> This was in the case of it being full screen, it would simply spill over onto the taskbar.

unfortunately when I try to take a screencap , the menu closes making it difficult to show.